src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js

```diff
@@ -57,30 +89,51 @@ const VersionsToolbar = ({
           {selectedVersion ? `v${selectedVersion.version}` : 'Select a version'}{' '}
           <Icon name="caret down" />
         </Menu.Item>
-
         <MuiMenu
           placeholder="Select a version"
           onClose={handleClose}
           open={Boolean(anchorEl)}
           anchorEl={anchorEl}>
-          {versionOptions.map(option => (
-            <MuiMenuItem
+          {versionOptions.map((option, index) => (
+            <StyledMenuItem
               key={option.key}
               selected={selectedVersion === option.value}
               onClick={e => {
                 onSelectVersion(e, { value: option.value })
                 handleClose()
               }}>
               <ListItemAvatar>
-                <Avatar>v{option.version.version}</Avatar>
+                <Avatar>{versionOptions.length - index}</Avatar>
               </ListItemAvatar>
               <ListItemText
-                primary={moment(option.version.createdAt).format(
-                  'MMM DD, YYYY, h:mm:ss a'
-                )}
-                secondary={option.version.status?.name}
+                primary={
+                  <div style={{ color: 'grey', width: 200 }}>Legal Start</div>
+                }
+                secondary={
+                  <div style={{ color: 'black' }}>
+                    {moment(option.version.effectiveLegalStart).format(
+                      'MMM DD YYYY h:mm a'
+                    )}
+                  </div>
+                }
+              />
+              <ListItemText
+                primary={
+                  <div style={{ color: 'grey', width: 200 }}>Legal End</div>
+                }
+                secondary={
+                  <div style={{ color: 'black' }}>
+                    {option.version.effectiveLegalEnd == null
+                      ? 'Present'
+                      : moment(option.version.effectiveLegalEnd).format(
+                          'MMM DD YYYY h:mm a'
+                        )}
+                  </div>
+                }
               />
-            </MuiMenuItem>
+              {option.version.status?.name === 'Approved' &&
+                createApprovedChip(option.version.effectiveLegalEnd == null)}
+            </StyledMenuItem>
```

Comment:
There are actually multiple statuses that can mean "approved", in the sense that the plan is legally binding. We're also only showing versions in this list that were legal at some point, so I think that means we'd end up showing "Approved" for every menu entry. The check you're doing will only match statuses that have the name explicitly of "Approved", but we have a couple others that *mean* approved but aren't *named* "Approved". I'm also not sure if displaying to the user that a version of plan was "approved" is important. @micheal-w-wells or @LisaMoore1, do you have any ideas on perhaps a more relevant piece of information to show here, or is this way alright?

This is an oversight on my part @okanji, I should have walked through what the statuses meant with you.